config/initializers/data_access_period.rb

```diff
@@ -0,0 +1 @@
+DATA_ACCESS_PERIOD_FOR_PUBLISHERS = 1.year.freeze
```

Comment:
DATA_ACCESS_PERIOD_FOR_PUBLISHERS needs to be available in both controllers and models now so I have added it to a data_access_period initializer.